htdocs/core/tpl/objectline_create.tpl.php

```diff
@@ -1123,8 +1123,8 @@ function(data) {
 				var multicurrency_code = $('option:selected', this).attr('data-multicurrency-code');                                			// When select is done from HTML select
 				if (multicurrency_code == undefined) { multicurrency_code = jQuery('#idprodfournprice').attr('data-multicurrency-code'); }  	// When select is done from HTML input with ajax autocomplete
 
-				var multicurrency_up = parseFloat($('option:selected', this).attr('data-multicurrency-unitprice'));                                	// When select is done from HTML select
-				if (isNaN(multicurrency_up)) { multicurrency_up = parseFloat(jQuery('#idprodfournprice').attr('data-multicurrency-unitprice')); }   // When select is done from HTML input with ajax autocomplete
+				var multicurrency_up = parseFloat($('option:selected', this).attr('data-multicurrency-up'));                                	// When select is done from HTML select
+				if (isNaN(multicurrency_up)) { multicurrency_up = parseFloat(jQuery('#idprodfournprice').attr('data-multicurrency-up')); }   // When select is done from HTML input with ajax autocomplete
```

Comment:
It seems the var name "multicurrency-up" is no more used anywhere in code in v20 branch.
So i suppose this has already been fixed by setting var name to "multicurrency-unitprice" everywhere.